src/GraphQL.Relay/GraphQL.Relay.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphQL" Version="5.2.0" />
+    <PackageReference Include="GraphQL" Version="5.3.1" />
```

Comment:
This should remain at 5.2.0 right?

Reply:
I would update. Remember that relay package is still in preview - 0.x.x , so we **can**.

Reply:
Makes sense